forge-game/src/main/java/forge/game/ability/ApiType.java

```diff
@@ -189,7 +189,7 @@ public enum ApiType {
     TapOrUntap (TapOrUntapEffect.class),
     TapOrUntapAll (TapOrUntapAllEffect.class),
     TimeTravel (TimeTravelEffect.class),
-    Token (TokenEffect.class, false),
+    Token (TokenEffect.class),
```

Comment:
@tool4ever if there are no more "non-stateless" Types, can the extra logic be removed?

Reply:
imo it doesn't hurt to keep it since it's a clean implementation
(should the need for it ever arise again)